### DISCUSSION TITLE
[BUG] Can't replicate neofetch exactly with same colors and text

### DISCUSSION THREAD

**Discussion by Scotty-Trees** [fastfetch-cli/fastfetch#1040] [Q&A] [answered]
Greetings and thank you for making fastfetch, such a great and fast utility, it shall replace neofetch for me once I can resolve these small issues. One issue I have is the neofetch colors I prefer are a bit lighter. I'm sure it's just a setting value I have to adjust, but I don't know how to do that exactly if you have any ideas I'd appreciate. Below is a screenshot of neofetch on top and fastfetch below it. You can see the top one has a lighter blue color for the font/text and that's the exact color I want. Fastfetch below has a darker blue color and I would love to know how/if I can change that to look just like the neofetch lighter blue color?

![Screenshot from 2024-06-24 11-48-58](https://github.com/fastfetch-cli/fastfetch/assets/45135187/1bdd5247-bfcb-4890-99af-673e47d74ce6)

The other issue I have is I always keep my Console terminal window at 83x26, but some of the text in fastfetch gets cutoff and I was wondering how/if I can adjust that so it doesn't display as much text. In the neofetch screenshot above, you can see some of the text doesn't extend very long, it was kept rather short and simple, which is what I am looking for. Below is another screenshot where you can see some of the text is getting cut off. Appreciate any help and thanks again for making this wonderful utility! 

![Screenshot from 2024-06-24 11-54-21](https://github.com/fastfetch-cli/fastfetch/assets/45135187/1928df9c-f8e2-4aec-86f0-0f2016f16ebb)

For context here it is compared to neofetch, where no text gets cutoff:

![Screenshot from 2024-06-24 12-01-48](https://github.com/fastfetch-cli/fastfetch/assets/45135187/e74ffc8b-bc71-4541-93cb-37a9111c4591)

I grabbed the neofetch.json from the presets page here, and not that I know what I'm doing, but I adjusted a few things, so mine currently looks like this btw: 

```
{
  "$schema": "https://github.com/fastfetch-cli/fastfetch/raw/dev/doc/json_schema.json",
  "modules": [
    "title",
    "separator",
    "os",
    "host",
    "kernel",
    "uptime",
    "packages",
    "shell",
    "display",
    "de",
    "wm",
    "wmtheme",
    "theme",
    "icons",
    "terminal",
    "cpu",
    "gpu",
    "memory",
    "break",
    "colors",
    "break",
    "break"
  ]
}
```

Edit1: Ok I did some more reading and editing (never coded anything before) and I got it closer to what I'm looking for, but I still can't replicate the color to get lighter and I'm wondering how can I change my display which reads: "Display (LM156LF1L03): 1920x1080" to just say "Display"? I don't need (LM156LF1L03) part of that at all. Here's my current updated code, which is probably messy, but I can't say I know what I'm doing other than trial and error:

```
{
  "$schema": "https://github.com/fastfetch-cli/fastfetch/raw/dev/doc/json_schema.json",
  "modules": [
    "title",
    "separator",
    {
            "type": "os",
            "format": "{3}"
        },
    "host",
    {
            "type": "kernel",
            "format": "{2}"
        },
    "uptime",
    "packages",
    "shell",
    {
            "type": "display",
            "format": "1920x1080"
        },
    "de",
    "wm",
    "wmtheme",
    "theme",
    "icons",
    "terminal",
      {
            "type": "cpu",
            "format": "Intel i5-8250U (8) @ 3.400GHz"
        },
    {
            "type": "gpu",
            "format": "{2}"
        },
    "memory",
    "break",
    "colors",
    "break",
    "break"
  ]
}
```
And here is the current look (shown below), almost there though, just need a few more touches, like getting it to show just "Display" only, getting the colors brighter and if possible I'd like to bring the margin of text on the right just a little closer to the left side as well if possible. 

![Screenshot from 2024-06-24 15-05-44](https://github.com/fastfetch-cli/fastfetch/assets/45135187/7df9cc11-fd9c-4d91-b2c7-89d92b4e8cff)

**Comment by CarterLi** [comment, marked as answer]
```jsonc
{
    "$schema": "https://github.com/fastfetch-cli/fastfetch/raw/dev/doc/json_schema.json",
    "display": {
        "size": {
            "maxPrefix": "MB",
            "ndigits": 0
        },
        "color": {
            "title": "38;5;12",
            "key": "38;5;12"
        }
    },
    "logo": {
        "source": "fedora",
        "color": {
            "1": "38;5;12"
        }
    },
    "modules": [
        "title",
        "separator",
        "os",
        "host",
        "kernel",
        "uptime",
        "packages",
        "shell",
        {
            "type": "display",
            "compactType": "original",
            "key": "Resolution"
        },
        "de",
        "wm",
        "wmtheme",
        "theme",
        "icons",
        "terminal",
        {
            "type": "terminalfont",
            "format": "{/2}{-}{/}{2}{?3} {3}{?}"
        },
        "cpu",
        {
            "type": "gpu",
            "key": "GPU"
        },
        {
            "type": "memory",
            "format": "{/1}{-}{/}{/2}{-}{/}{} / {}"
        },
        "break",
        "colors"
    ]
}
```

**Reply by Scotty-Trees** [reply]
My apologies I didn't see this reply the other day, but thank you for getting back so quickly and helping out, I really appreciate your assistance. Now my neofetch transition is complete, so thank you very much again for all the hard work you do.
